connectivity/FEATURE_BLE/source/generic/FileSecurityDb.h

```diff
@@ -136,7 +136,7 @@ class FileSecurityDb : public SecurityDb {
 
     virtual void restore();
 
-    virtual void sync(entry_handle_t db_handle);
+    virtual void sync(entry_handle_t db_handle = invalid_entry_handle);
```

Comment:
What do we gain with the default parameter ? It is an implementation and it won't be called directly by the BLE framework.

Reply:
So that I don't have to modify existing code.